tests/components/script/test_init.py

```diff
@@ -1502,3 +1502,33 @@ async def test_blueprint_script_fails_substitution(
         "{'service_to_call': 'test.automation'}: No substitution found for input blah"
         in caplog.text
     )
+
+
+async def test_responses(hass: HomeAssistant) -> None:
+    """Test we can get responses."""
+    mock_restore_cache(hass, ())
+    assert await async_setup_component(
+        hass,
+        "script",
+        {
+            "script": {
+                "test": {
+                    "sequence": {
+                        "stop": "done",
+                        "response": "5",
```

Comment:
Responses should be a dictionary right?

Reply:
Well, so that is an interesting thought ! Should we force our users to return dictionaries  from their scripts or should we wrap it?

Note that for the WebSocket API there is no such limitation, as the `execute_script` command interacts directly with the Script helper

Reply:
> Should we force our users to return dictionaries from their scripts or should we wrap it?

I think we should? As in, it was part of the design/specifications that services must return a JSON serializable dictionary/response. This moves away from that for scripts, which is a bit odd / potentially unexpected by consumers.

Reply:
Thought about it, I agree. Consistency wins.